recipe/conda_build_config.yaml

```diff
@@ -272,9 +272,9 @@ aws_crt_cpp:
 aws_sdk_cpp:
   - 1.9.160
 boost:
-  - 1.74.0
+  - 1.78.0
 boost_cpp:
-  - 1.74.0
+  - 1.78.0
```

Comment:
I think historically we temporarily supported 2 versions of boost since it's a pretty widely used dependency that isn't the easiest to always migrate. Do we not want to do that again?

Reply:
That was only during the migration IIRC?

Reply:
As far as I remember, we only wanted to keep two versions active during the migration due to its length. Once we have finished the migration, most of the maintained packages should be happily solvable.

Reply:
Got it, thanks all! Apologies for the noise.